retrofit2/src/main/java/com/linecorp/armeria/client/retrofit2/RetrofitMeterIdPrefixFunction.java

```diff
@@ -107,7 +107,7 @@ private void buildTags(ImmutableList.Builder<Tag> tagListBuilder, RequestOnlyLog
         if (invocation != null) {
             if (serviceTagName != null) {
                 final String service = firstNonNull(serviceName,
-                                                    invocation.method().getDeclaringClass().getSimpleName());
+                                                    invocation.method().getDeclaringClass().getName());
```

Comment:
Should we now default `serviceTagName` to `"service"`? Or do we already?

Reply:
We set `serviceTagName` to `"service"` conditionally.
- If `serviceName` is null, `serviceTagName` could be nullable. (Users might not be interested in the `service` tag)
- If `serviceName` is not null, the default name of `serviceTagName` is `"service"`

https://github.com/line/armeria/blob/2a48e9aa0a5d1b5b82ffe9755b9d2e1d3b96b137/retrofit2/src/main/java/com/linecorp/armeria/client/retrofit2/RetrofitMeterIdPrefixFunction.java#L80-L84

Reply:
IIRC it was due to backward compatibility. Perhaps it's time to always add `service` by default? /cc @kojilin @arhont375